driver-http/src/main/resources/activities/baselines/http-iot.yaml

```diff
@@ -27,8 +27,6 @@ bindings:
 blocks:
   - tags:
       phase: schema
-    params:
-      prepared: false
```

Comment:
This params property sets the default for any statements in this block, which is only DDL. If the statements are prepared (true by defult), then they will be rejected, as prepared statements need a schema to be checked against.

Reply:
Maybe we can find a better way to handle this in terms of defaults and user feedback.

Reply:
Can we close this PR and discuss the issue in more detail?

Reply:
yes plz